src/test/ui/consts/const_limit/const_limit_not_reached.rs

```diff
@@ -0,0 +1,15 @@
+// run-pass
+#![feature(const_limit)]
+#![const_limit="1000"]
```

Comment:
I still think it would be good to have a test with a low limit that will fail to compile because the evaluation hits the limit.